thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/api/NamespaceConfigurationApi.java

```diff
@@ -18,7 +18,10 @@ public class NamespaceConfigurationApi implements ThirdEyeCrudApi<NamespaceConfi
   private Long id;
   private AuthorizationConfigurationApi auth;
 
-  private TimeConfigurationApi timeConfiguration;
+  private TimeConfigurationApi timeConfiguration = new TimeConfigurationApi();
```

Comment:
why are we using default values in api object? It should always come from db. right? 
we already have default values in config objects

Reply:
indeed, just the result of me trying things removing this